### PR TITLE
plantalytics-frontend#22

### PR DESCRIPTION
**Pull request by SoySauceLabs**
For testing, please follow the **How to Run Locally** section of the README.md file. The acceptance criteria are:
- [x] Observing the "Hello, World!" message detailed in the README.md file.
- [x] Receiving a 200 response code in the terminal window that's running the server.

Check it out!!

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+.idea
+src/.idea
```
Comment (scotte216) [inline .gitignore:2]: Should include .pyc files in .gitignore

File: src/.idea/encodings.xml [hunk 2]
```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="Encoding">
-    <file url="PROJECT" charset="UTF-8" />
-  </component>
-</project>
```
Comment (scotte216) [inline src/.idea/encodings.xml]: need a blank line at the end of file.

File: src/.idea/misc.xml [hunk 3]
```diff
@@ -1,14 +0,0 @@
...
-    <OptionsSetting value="true" id="Remove" />
-    <OptionsSetting value="true" id="Checkout" />
-    <OptionsSetting value="true" id="Update" />
-    <OptionsSetting value="true" id="Status" />
-    <OptionsSetting value="true" id="Edit" />
-    <ConfirmationsSetting value="0" id="Add" />
-    <ConfirmationsSetting value="0" id="Remove" />
-  </component>
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.5.1 (C:\Users\Matt\AppData\Local\Programs\Python\Python35\python.exe)" project-jdk-type="Python SDK" />
-</project>
```
Comment (scotte216) [inline src/.idea/misc.xml]: Need blank line at EOF

File: src/.idea/modules.xml [hunk 4]
```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectModuleManager">
-    <modules>
-      <module fileurl="file://$PROJECT_DIR$/.idea/src.iml" filepath="$PROJECT_DIR$/.idea/src.iml" />
-    </modules>
-  </component>
-</project>
```
Comment (scotte216) [inline src/.idea/modules.xml]: Need blank line EOF

File: src/dummy/urls.py [hunk 5]
```diff
@@ -0,0 +1,6 @@
+from django.conf.urls import url
+from .import views
+
+urlpatterns = [
+    url(r'^$', views.index, name='index'),
+]
```
Comment (scotte216) [inline src/dummy/urls.py:6]: Include blank line at EOF.

File: README.md [hunk 6]
```diff
@@ -8,7 +8,7 @@ This project is licensed under the MIT License. Please see the file LICENSE in t
 
 ## How to Run Locally
 
-So you'd like to give this project a spin? You've in luck, follow these 
+So you'd like to give this project a spin? You've in luck, follow these
```
Comment (scotte216) [inline README.md]: "You've" should be "You're"

File: .gitignore (hunk 1 above)
Comment (SoySauceLabs) [inline .gitignore:2]: Added!

File: src/dummy/urls.py (hunk 5 above)
Comment (SoySauceLabs) [inline src/dummy/urls.py:6]: Added!

File: README.md (hunk 6 above)
Comment (SoySauceLabs) [inline README.md]: Good catch, corrected!

**Pull request merged**